catalog-info.yaml

```diff
@@ -0,0 +1,30 @@
+# Backstage documentation
+# https://backstage.io/docs/features/software-catalog/descriptor-format/
+
+# !!! WARNING !!!
+# This is a template file with a number of fields that need to be filled before merging this to the default branch
+apiVersion: backstage.io/v1alpha1
+# Component, API, Template, Group, User, Resource, System, Domain, Location
+kind: unknown
+metadata:
+  name: forma-36 
+  description: unknown
+  annotations:
+    github.com/project-slug: contentful/forma-36 
+    contentful.com/service-tier: "unknown" #1, 2, 3, 4
```

Comment:
@roryscarson this is a front-end only, should we leave `"unknown"` in this case?

Reply:
If it doesn't have an SLO associated with it then it's tier-4. https://contentful.atlassian.net/wiki/spaces/ENG/pages/2275311671/Services+Tiers+v2